components/org.wso2.carbon.identity.conditional.auth.functions.entgra/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/entgra/GetDeviceInfoEntgraFunctionImpl.java

```diff
@@ -266,4 +287,62 @@ private JSONObject getErrorJsonObject(Constants.AuthResponseErrorCode errorCode,
         errorMap.put("errorMessage", errorMessage);
         return errorMap;
     }
+
+    /**
+     * Return true if the request domain is valid or return false if the request domain is invalid.
+     * @param url
+     * @return validity
+     */
+    private boolean isValidRequestDomain(URI url) {
```

Comment:
Isn't there any util method for this?

Reply:
I got this from the [http conditional auth function](https://github.com/wso2-extensions/identity-conditional-auth-functions/blob/master/components/org.wso2.carbon.identity.conditional.auth.functions.http/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/http/AbstractHTTPFunction.java). It was implemented as a private method and I didn't find any util method for this.